parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -59,7 +59,18 @@ installStandaloneContainerd() {
     if [[ -f /etc/containerd/config.toml.rpmsave ]]; then
         mv /etc/containerd/config.toml.rpmsave /etc/containerd/config.toml
     fi
+    installMissingMobyComponentsForContainerd
+}
 
+installMissingMobyComponentsForContainerd() {
+    # install moby components common between containerd and docker
+    local MOBY_VERSION="19.03.14"
+    MOBY_CLI=${MOBY_VERSION}
+    if [[ "${MOBY_CLI}" == "3.0.4" ]]; then
+        MOBY_CLI="3.0.3"
```

Comment:
err what?

Reply:
also don't think we really need this for mariner, it is mostly to avoid breaking older ubuntu users, you can no op it or fail as unimplemented